### DISCUSSION TITLE
beatmap downloads on local build?

### DISCUSSION THREAD

**Discussion by 1vers1on** [ppy/osu#19806] [Ideas] [answered]
Since testflight is full I just built lazer for my phone. I am unable to download beatmaps or login because it is a local build. I understand if score submission is disabled due to cheating, but I don't see any reason my login / beatmap download is disabled.

**Comment by peppy** [comment, marked as answer]
you're on your own in figuring out why, but nothing is disabled.

**Comment by naipofo** [comment]
Does your build connect to the development server? If so, your account won't work there, you might need to create a new one.

**Reply by 1vers1on** [reply]
What's the development server?

**Reply by naipofo** [reply]
https://dev.ppy.sh/ 
It's a mirror of the main server, meant for testing. All the beatmaps are still there, but scores/users are separate. If you just clone the repo and build, it defaults to the dev server.

**Reply by 1vers1on** [reply]
Got the normal server working. I had to set `UseDevelopmentServer` to false.
